docs/automated_testing.md

```diff
@@ -991,38 +993,162 @@ uses: suffolkLITLab/ALKiln@releases/v4
 
 When you want to update to a new version of the ALKiln, update that sha manually.
 
+### Set ALKiln's npm version
+
+This one requires prior technical knowledge. Feel free to ask us.
+
+You can use an exact npm version of ALKiln by using your workflow file's `ALKILN_VERSION` input. The default uses a carat, for example `^4.0.0`. That means it will use the latest minor or patch in version 4 of ALKiln. You can instead use a specific version, for example `4.3.0`. See our section on [setting optional inputs](#optional-inputs).
+
 ### GitHub secrets
 
-Use GitHub secrets to set variable values with sensitive information. For example, a password. The value of this variable will not appear anywhere in the report or in the console. You also will be unable to take a screenshot of the page.
+<!-- TODO: Rearrange - Maybe use this section to refer to another section that is more focused on setting arbitrary environment variables in general -->
+
+You can use GitHub secrets to set environment variable values with sensitive information. For example, a password. The value of this variable will not appear anywhere in the report or in the console. You also will be unable to take a screenshot of the page and ALKiln will avoid taking an error screenshot.
```

Comment:
A think I'm realizing is that most people don't know what environment variables are. Can be addressed later though.

Reply:
Link GitHub docs about env vars where they're mentioned.